HW3/src/js/task2.js

```diff
@@ -1,22 +1,18 @@
-let firstLengthSide = prompt('Enter a length of side');
-let secondLengthSide = prompt('Enter b length of side');
+let a = parseFloat(prompt('Enter length of side A'));
```

Comment:
rename variable cause it's hard to understand what's mean a & b

Reply:
check pull request for 3 HW not this